src/listeners/onVoiceChannelJoin/jesus.ts

```diff
@@ -0,0 +1,25 @@
+import { Message, SnowflakeUtil, TextChannel, VoiceState } from "discord.js";
+import commands from "../../commands";
+
+const JESUS_MEMBER_ID = "191931417314328576";
+
+export default (oldState: VoiceState, newState: VoiceState) => {
+    if (newState.member?.id !== JESUS_MEMBER_ID) return;
+
+    const client = newState.guild.client;
+    const sudoChannel = newState.guild.channels.cache.find((c) => c.name === "sudo" && c.type === "text") as TextChannel;
+
+    if (sudoChannel) {
```

Comment:
Same as above, it's probably cleaner to early return instead of wrapping the method content in the `if`